plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityDomestication.java

```diff
@@ -0,0 +1,44 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.entity.AbstractHorse;
+
+public class EntityDomestication extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name domestication
+    // @input ElementTag(Number)
+    // @description
+    // Controls how close to being fully tamed a horse-type entity is.
+    // Value must be an integer between 1 and 2,147,483,647 inclusive.
```

Comment:
`and 2,147,483,647 inclusive.`
_what_? don't just... put int32 max there, that's unhinged.

I'm guessing the actual max value is, maybe, yknow, the max_domestication property??

Also why does 1 mean? what does the max value mean?

What in-game actions normally change this value?

Does the value correspond to a number of ticks, a number of times a player has done an action, ...?